eng/common/TestResources/New-TestResources.ps1

```diff
@@ -393,6 +393,17 @@ foreach ($templateFile in $templateFiles) {
                 $DebugPreference = "Continue"
             }
             New-AzResourceGroupDeployment -Name $BaseName -ResourceGroupName $resourceGroup.ResourceGroupName -TemplateFile $templateFile -TemplateParameterObject $templateFileParameters
+        } catch {
+            $msg = @(
+                "",
+                "==================================================",
+                "For help debugging live test provisioning issues,",
+                "see http://aka.ms/azure-sdk-live-test-help",
+                "=================================================="
+            ) -join ([Environment]::NewLine)
```

Comment:
While this approach works and allows you to properly indent you could also consider using [here strings])(https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_quoting_rules?view=powershell-7.1#here-strings)

Reply:
I chose against using a here string because of the indentation issue. I'm not too opinionated either way, so I'm happy to change it if you prefer the other way?